service/frontend/workflowHandler.go

```diff
@@ -3399,3 +3392,42 @@ func (wh *WorkflowHandler) makeFakeContinuedAsNewEvent(
 		},
 	}, nil
 }
+
+func (wh *WorkflowHandler) validateNamespace(
```

Comment:
we don't need to do namespace check. Namespace is validated by NamespaceValidatorInterceptor

Reply:
ignore this, we do need this validation for register namespace.